### PR TITLE
[AllApps] Removing `__future__`

### PR DESCRIPTION
**Pull request by sunethwarna**
**📝 Description**
As in the title. This removes support for python2.

**🆕 Changelog**
- Removes `__future__`

### CODE REVIEW

File: applications/CableNetApplication/CableNetApplication.py [hunk 1]
```diff
@@ -1,5 +1,4 @@
 # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/CableNetApplication/CableNetApplication.py]: > 
```suggestion
```

File: applications/ChimeraApplication/ChimeraApplication.py [hunk 2]
```diff
@@ -1,5 +1,4 @@
 # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ChimeraApplication/ChimeraApplication.py]:
```suggestion
```

File: applications/CompressiblePotentialFlowApplication/CompressiblePotentialFlowApplication.py [hunk 3]
```diff
@@ -1,5 +1,4 @@
 # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/CompressiblePotentialFlowApplication/CompressiblePotentialFlowApplication.py]:
```suggestion
```

File: applications/ConstitutiveModelsApplication/ConstitutiveModelsApplication.py [hunk 4]
```diff
@@ -1,5 +1,4 @@
 # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ConstitutiveModelsApplication/ConstitutiveModelsApplication.py]:
```suggestion
```

File: applications/ConvectionDiffusionApplication/ConvectionDiffusionApplication.py [hunk 5]
```diff
@@ -1,5 +1,4 @@
 # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ConvectionDiffusionApplication/ConvectionDiffusionApplication.py]:
```suggestion
```

File: applications/DamApplication/DamApplication.py [hunk 6]
```diff
@@ -1,5 +1,4 @@
 # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/DamApplication/DamApplication.py]:
```suggestion
```

File: applications/ExaquteSandboxApplication/ExaquteSandboxApplication.py [hunk 7]
```diff
@@ -1,5 +1,4 @@
 # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ExaquteSandboxApplication/ExaquteSandboxApplication.py]:
```suggestion
```

File: applications/FreeSurfaceApplication/FreeSurfaceApplication.py [hunk 8]
```diff
@@ -1,5 +1,4 @@
 # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/FreeSurfaceApplication/FreeSurfaceApplication.py]:
```suggestion
```

File: applications/MultilevelMonteCarloApplication/MultilevelMonteCarloApplication.py [hunk 9]
```diff
@@ -1,5 +1,4 @@
 # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/MultilevelMonteCarloApplication/MultilevelMonteCarloApplication.py]:
```suggestion
```

File: applications/PfemFluidDynamicsApplication/PfemFluidDynamicsApplication.py [hunk 10]
```diff
@@ -1,5 +1,4 @@
 #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/PfemFluidDynamicsApplication/PfemFluidDynamicsApplication.py]:
```suggestion
```

File: applications/PfemFluidDynamicsApplication/python_scripts/python_solvers_wrapper_pfem_fluid.py [hunk 11]
```diff
@@ -1,5 +1,4 @@
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/PfemFluidDynamicsApplication/python_scripts/python_solvers_wrapper_pfem_fluid.py]:
```suggestion
```

File: applications/PoromechanicsApplication/PoromechanicsApplication.py [hunk 12]
```diff
@@ -1,5 +1,4 @@
 # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/PoromechanicsApplication/PoromechanicsApplication.py]:
```suggestion
```

File: applications/RomApplication/RomApplication.py [hunk 13]
```diff
@@ -1,5 +1,4 @@
 # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/RomApplication/RomApplication.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/algorithms/algorithm_base.py [hunk 14]
```diff
@@ -10,7 +10,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/algorithms/algorithm_base.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/algorithms/algorithm_bead_optimization.py [hunk 15]
```diff
@@ -10,7 +10,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/algorithms/algorithm_bead_optimization.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/algorithms/algorithm_factory.py [hunk 16]
```diff
@@ -9,7 +9,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/algorithms/algorithm_factory.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/algorithms/algorithm_gradient_projection.py [hunk 17]
```diff
@@ -10,7 +10,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/algorithms/algorithm_gradient_projection.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/algorithms/algorithm_penalized_projection.py [hunk 18]
```diff
@@ -10,7 +10,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/algorithms/algorithm_penalized_projection.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/algorithms/algorithm_steepest_descent.py [hunk 19]
```diff
@@ -10,7 +10,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/algorithms/algorithm_steepest_descent.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/algorithms/algorithm_trust_region.py [hunk 20]
```diff
@@ -9,7 +9,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/algorithms/algorithm_trust_region.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/analyzers/analyzer_base.py [hunk 21]
```diff
@@ -9,7 +9,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/analyzers/analyzer_base.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/analyzers/analyzer_empty.py [hunk 22]
```diff
@@ -9,7 +9,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/analyzers/analyzer_empty.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/analyzers/analyzer_factory.py [hunk 23]
```diff
@@ -9,7 +9,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/analyzers/analyzer_factory.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/communicator_factory.py [hunk 24]
```diff
@@ -9,7 +9,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/communicator_factory.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/loggers/data_logger_factory.py [hunk 25]
```diff
@@ -10,7 +10,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/loggers/data_logger_factory.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/loggers/design_logger_base.py [hunk 26]
```diff
@@ -10,7 +10,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/loggers/design_logger_base.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/loggers/design_logger_gid.py [hunk 27]
```diff
@@ -10,7 +10,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/loggers/design_logger_gid.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/loggers/design_logger_unv.py [hunk 28]
```diff
@@ -10,7 +10,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/loggers/design_logger_unv.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/loggers/design_logger_vtk.py [hunk 29]
```diff
@@ -10,7 +10,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/loggers/design_logger_vtk.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/mapper_factory.py [hunk 30]
```diff
@@ -9,7 +9,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/mapper_factory.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/mesh_controllers/mesh_controller_base.py [hunk 31]
```diff
@@ -9,7 +9,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/mesh_controllers/mesh_controller_base.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/mesh_controllers/mesh_controller_basic_updating.py [hunk 32]
```diff
@@ -9,7 +9,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/mesh_controllers/mesh_controller_basic_updating.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/mesh_controllers/mesh_controller_with_solver.py [hunk 33]
```diff
@@ -9,7 +9,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/mesh_controllers/mesh_controller_with_solver.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/model_part_controller_factory.py [hunk 34]
```diff
@@ -9,7 +9,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/model_part_controller_factory.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/optimizer_factory.py [hunk 35]
```diff
@@ -9,7 +9,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/optimizer_factory.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/utilities/custom_math.py [hunk 36]
```diff
@@ -9,7 +9,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/utilities/custom_math.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/python_scripts/utilities/custom_timer.py [hunk 37]
```diff
@@ -10,7 +10,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/python_scripts/utilities/custom_timer.py]:
```suggestion
```

File: applications/ShapeOptimizationApplication/tests/test_ShapeOptimizationApplication.py [hunk 38]
```diff
@@ -3,7 +3,6 @@
 # ==============================================================================
 
 # Making KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/ShapeOptimizationApplication/tests/test_ShapeOptimizationApplication.py]:
```suggestion
```

File: applications/SolidMechanicsApplication/python_scripts/solid_analysis.py [hunk 39]
```diff
@@ -1,5 +1,4 @@
 # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/SolidMechanicsApplication/python_scripts/solid_analysis.py]:
```suggestion
```

File: applications/UmatApplication/UmatApplication.py [hunk 40]
```diff
@@ -1,5 +1,4 @@
 # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
```
Comment (sunethwarna) [inline applications/UmatApplication/UmatApplication.py]:
```suggestion
```

**Review by rubenzorrilla — APPROVED** [review]
I went through all the files and changes are correct. Thanks for the effort.

**Pull request merged**